.github/workflows/end2end.yaml

```diff
@@ -140,16 +140,6 @@ jobs:
           mkdir -p /tmp/artifacts/data/${{ github.job }}/act-logs/
           mv tests/workflows/act-*.log /tmp/artifacts/data/check-workflows/act-logs/
           tar zcvf /tmp/artifacts/${{ github.sha }}-${{ github.job }}-act-logs.tgz /tmp/artifacts/data/${{ github.job }}/act-logs;
-      - name: Set status in ARTIFACTS to enable re-run only failed jobs
```

Comment:
IIRC, in the release workflow, `scality/action-artifacts@v3`, we rely on this before creating the promotion of the artifacts. Will it work if we remove it from the build?

Reply:
The status is still written at the end of the build (either the `write-final-failed-status` or `write-final-success-status`), so we should be fine.

(this was discussed in https://scality.slack.com/archives/CFDJAPQ6S/p1709823455660169)